pom.xml

```diff
@@ -957,6 +957,9 @@
               <systemId>src/main/resources/com/puppycrawl/tools/checkstyle/configuration_1_3.dtd</systemId>
             </validationSet>
           </validationSets>
+          <catalogs>
+            <catalog>config/catalog.xml</catalog>
+          </catalogs>
```

Comment:
@romani 
I made it as a separate commit and checked that this approach really works without Internet connection.